rfc-013-code-repositories.md

```diff
@@ -0,0 +1,89 @@
+# Code repository standards
+
+## Summary
+
+A set of standards to be used for projects that will be shared within Made Tech or the greater technology community. These are projects hosted on GitHub, Bitbucket, Gitlab or similar public source control management.
+
+## Problem
+
+Made Tech code repositories (to be referred to as repos herein) do not have a common look and feel.
+
+## Visibility
+
+The visibility of your project **SHOULD** mean the following:
+
+- You **MUST** set your repo to **Public** unless you have a reason not to.
+- You **MUST** set your repo to **Private to the Customers organisation** if the customer owns the Intellectual Property and has a public source control management platform account.
+- You **MUST** set your repo to **Private to the Made Tech organisation** if the customer owns the Intellectual Property and has a public source control management platform account.
+
+## Files in the Repository
+
+### README.md
+
+The `README.md` **MUST** exist for all repos.
+
+#### Headings
+
+The following headings **SHOULD** be considered the absolute minimum to include in the `README.md` when making a project public.
```

Comment:
Why only public projects?  These feel applicable for private too?

Reply:
Ah missed that, after discussion with others we decided it was mandatory for all repos.